app/pb_kits/playbook/pb_contact/_contact.jsx

```diff
@@ -26,15 +27,24 @@ const Contact = ({
   className,
   dark=false,
   contactValue,
+  contactDetail,
 }: ContactProps) => {
 
+  const formatDetail = (contactDetail) => {
+    if (contactDetail !== undefined) {
+      return `\u00b7 ${contactDetail}`
+    } else {
+      return ""
+    }
+  }
+
```

Comment:
```suggestion
const formatDetail = (contactDetail) => {
    if (contactDetail !== undefined) {
      return `\u00b7 ${contactDetail}`
    } else {
      return contactDetail
    }
}
```

Reply:
Hey @thestephenmarshall, is this just a suggested indentation change?  This `const` is the first in the `ContactProps` hash, so they're all indented. Let me know if something still needs adjusting here. Thanks!